pyglet_demo/main.py

```diff
@@ -23,19 +25,24 @@ def on_draw():
         window.clear()
 
         game_app.on_draw()
-    
+
     @window.event
     def on_mouse_press(x, y, button, modifiers):
         game_app.on_click(x, y, button)
-    
+
     inputHandler = cameraInput.Input(game_app.spin)
+    # dev board will need to be changed. not sure of the name in debian
+    serialInputHandler = serialInput.Input(
+        '/dev/tty.usbmodem1101', game_app.spin)
```

Comment:
This line will cause errors if the receiver Adafruit feather is not connected. Will have to adjust based on how the ports show up on the Jetson

Reply:
Go ahead and make the port /dev/feather.  I'll add an issue to make it appear on the same devices every time.